Package react-native-webview has been ignored because it contains invalid configuration

<!---
Hello and thank you for taking the time to report a bug.
This is a community repository, meaning that it's everyone's responsibility to address bugs and send pull requests.
Maintainers will be here to help/guide you if you have questions.
Please search in the opened issues before submitting a new one.
Issues with no activity for a long time will be closed.
--->
**Bug description:**
<!---
A clear and concise description of what the bug is.
--->
I am trying to use react-native-webview and following step. But it doesn't work.

yarn add react-native-webview
cd ios && pod install && cd .. # CocoaPods on iOS needs this extra step

There is problem.
![image](https://user-images.githubusercontent.com/40887236/161056468-9460e0fd-3d44-4788-8c8b-7141f0df4be5.png)


**To Reproduce:**
<!---
Here you should put a link to a repository with a reproduction of the bug
Please also describe the steps required for the bug to happen
--->

**Expected behavior:**
<!---
A clear and concise description of what you expected to happen.
--->

**Screenshots/Videos:**
<!---
If applicable, add screenshots to help explain your problem.
--->

**Environment:**
 - OS: macOS
 - OS version: 12.3
 - react-native version: 0.67.4
 - react-native-webview version: 11.18.0

same problem for me on macOS 
```js
arn run v1.22.17
$ react-native run-android
warn Package react-native-webview has been ignored because it contains invalid configuration. Reason: Cannot find module 'react-native-test-app/package.json'
Require stack:
- /Users/midknight/epitech/area/mobile-app/node_modules/react-native-webview/react-native.config.js
- /Users/midknight/epitech/area/mobile-app/node_modules/cosmiconfig/node_modules/import-fresh/index.js
- /Users/midknight/epitech/area/mobile-app/node_modules/cosmiconfig/dist/loaders.js
- /Users/midknight/epitech/area/mobile-app/node_modules/cosmiconfig/dist/createExplorer.js
- /Users/midknight/epitech/area/mobile-app/node_modules/cosmiconfig/dist/index.js
- /Users/midknight/epitech/area/mobile-app/node_modules/@react-native-community/cli/build/tools/config/readConfigFromDisk.js
- /Users/midknight/epitech/area/mobile-app/node_modules/@react-native-community/cli/build/tools/config/index.js
- /Users/midknight/epitech/area/mobile-app/node_modules/@react-native-community/cli/build/commands/install/install.js
- /Users/midknight/epitech/area/mobile-app/node_modules/@react-native-community/cli/build/commands/index.js
- /Users/midknight/epitech/area/mobile-app/node_modules/@react-native-community/cli/build/index.js
- /Users/midknight/epitech/area/mobile-app/node_modules/react-native/cli.js
info Running jetifier to migrate libraries to AndroidX. You can disable it using "--no-jetifier" flag.
Jetifier found 1017 file(s) to forward-jetify. Using 10 workers...
info JS server already running.
info Installing the app...
```

--- a/package.json
+++ b/package.json
@@ -64,7 +64,7 @@
     "react": "17.0.2",
     "react-native": "^0.66.0-0",
     "react-native-macos": "^0.66.0-0",
-    "react-native-test-app": "^1.1.2",
+    "react-native-test-app": "^1.3.2",
     "react-native-windows": "^0.66.0-0",
     "selenium-appium": "1.0.2",
     "selenium-webdriver": "4.0.0-alpha.7",

--- a/react-native.config.js
+++ b/react-native.config.js
@@ -1,13 +1,32 @@
-const fs = require('fs');
-const path = require('path');
-
-const windowsProjectFile = path.join(
-  'node_modules',
-  '.generated',
-  'windows',
-  'ReactTestApp',
-  'ReactTestApp.vcxproj',
-);
+const project = (() => {
+  const fs = require('fs');
+  const path = require('path');
+  try {
+    const {
+      androidManifestPath,
+      iosProjectPath,
+      windowsProjectPath,
+    } = require('react-native-test-app');
+    return {
+      android: {
+        sourceDir: path.join('example', 'android'),
+        manifestPath: androidManifestPath(
+          path.join(__dirname, 'example', 'android'),
+        ),
+      },
+      ios: {
+        project: iosProjectPath('example/ios'),
+      },
+      windows: fs.existsSync('example/windows/WebviewExample.sln') && {
+        sourceDir: path.join('example', 'windows'),
+        solutionFile: 'WebviewExample.sln',
+        project: windowsProjectPath(path.join(__dirname, 'example', 'windows')),
+      },
+    };
+  } catch (_) {
+    return undefined;
+  }
+})();
 
 module.exports = {
   dependencies: {
@@ -30,51 +49,5 @@ module.exports = {
       },
     },
   },
-  project: {
-    android: {
-      sourceDir: path.join('example', 'android'),
-      manifestPath: path.relative(
-        path.join(__dirname, 'example', 'android'),
-        path.join(
-          path.dirname(require.resolve('react-native-test-app/package.json')),
-          'android',
-          'app',
-          'src',
-          'main',
-          'AndroidManifest.xml',
-        ),
-      ),
-    },
-    ios: {
-      project: (() => {
-        const {
-          packageSatisfiesVersionRange,
-        } = require('react-native-test-app/scripts/configure');
-        if (
-          packageSatisfiesVersionRange(
-            '@react-native-community/cli-platform-ios',
-            '<5.0.2',
-          )
-        ) {
-          // Prior to @react-native-community/cli-platform-ios v5.0.0,
-          // `project` was only used to infer `sourceDir` and `podfile`.
-          return 'example/ios/ReactTestApp-Dummy.xcodeproj';
-        }
-        // `sourceDir` and `podfile` detection was fixed in
-        // @react-native-community/cli-platform-ios v5.0.2 (see
-        // https://github.com/react-native-community/cli/pull/1444).
-        return 'node_modules/.generated/ios/ReactTestApp.xcodeproj';
-      })(),
-    },
-    windows: fs.existsSync(windowsProjectFile) && {
-      sourceDir: path.join('example', 'windows'),
-      solutionFile: 'WebviewExample.sln',
-      project: {
-        projectFile: path.relative(
-          path.join(__dirname, 'example', 'windows'),
-          windowsProjectFile,
-        ),
-      },
-    },
-  },
+  ...(project ? { project } : undefined),
 };

